compiler/plugins/target/AMD-AIE/Translation/AIESerializer.cpp

```diff
@@ -516,7 +516,7 @@ LogicalResult AccelSerializer::processOperation(Operation *opInst,
 
   auto status =
       TypeSwitch<Operation *, LogicalResult>(opInst)
-          .Case<func::FuncOp, scf::ForallOp, scf::ForOp,
+          .Case<mlir::FunctionOpInterface, scf::ForallOp, scf::ForOp,
```

Comment:
Any way we could skip having to specify the `mlir::` namespace here for `FunctionOpInterface` ?
Just how it is done in the same PR in [AMDAIELowerExecutableTarget.cpp](https://github.com/nod-ai/iree-amd-aie/blob/e9fc58296903ae570474d3890d4395d5733d878b/compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIELowerExecutableTarget.cpp#L70).